scripts/review-kredits.js

```diff
@@ -121,6 +136,17 @@ async function initializeKredits () {
   return kredits;
 }
 
+function createContribution(contributorName, contributionAttributes, Contribution) {
+  console.log(`Creating contribution token for ${contributionAttributes.amount}₭S to ${contributorName} for ${contributionAttributes.description}...`);
+
+  return Contribution.add(contributionAttributes).catch(error => {
```

Comment:
why do you pass in `Contribution` here and not use `kredits.Contribution` from the global variable?

Reply:
`kredits` is not global.

Reply:
oh true. 
though I guess it could be?